### DISCUSSION TITLE
Record update (tweet)

### DISCUSSION THREAD

**Discussion by ggreif** [dfinity/motoko#4808] [Show and tell]
As Motoko-coded canisters are proper databases, you often want to perform table joins.

You can use _functional record updates_ to join and modify objects:
``` Motoko
{ employee[n] and employee[n].dept with timestamp = now() }`
```
This will result in the timestamped joined fields of the given employee with its department.

Read more at: https://internetcomputer.org/docs/current/motoko/main/reference/language-manual#object-combinationextension.

**Comment by jessiemongeon1** [comment]
Scheduled for Dev Twitter post on 4/4
